modules/RTC/JitsiLocalTrack.js

```diff
@@ -28,26 +29,92 @@ function JitsiLocalTrack(stream, videoType,
             this.dontFireRemoveEvent = false;
         }.bind(this));
     this.initialMSID = this.getMSID();
+    this.inMuteOrUnmuteProcess = false;
 }
 
 JitsiLocalTrack.prototype = Object.create(JitsiTrack.prototype);
 JitsiLocalTrack.prototype.constructor = JitsiLocalTrack;
 
 /**
+ * Mutes the track. Will reject the Promise if there is mute/unmute operation
+ * in progress.
+ * @returns {Promise}
+ */
+JitsiLocalTrack.prototype.mute = function () {
```

Comment:
Looks good. You could have made it part of the prototype to save you form having to do the context binding.